src/frontend/tsx/GamePanel.tsx

```diff
@@ -161,59 +162,61 @@ export function GamePanel () {
         </div>
         {/* buttons */}
         {/* resign button */}
-        <button
-          className='option'
-          disabled={!hasGame}
-          onClick={resignOnClick}>
-          Resign</button>
+        {!activeGame ? (
+          <button
+            className='option'
+            disabled={!activeGame}
+            onClick={resignOnClick}>
+           Resign</button>
+        ) : (null)
+        }
```

Comment:
Remove the `!activeGame ?` wrapper for now. Controlling whether gameplay buttons are visible should be done: a) altogether; b) in a separate PR.